libs/live-network/src/network.ts

```diff
@@ -175,6 +175,10 @@ export type LiveNetworkResponse<T> = {
   data: T;
   status: number;
 };
+
+// inspired by https://github.com/sindresorhus/got/blob/main/documentation/7-retry.md#retry
+const retryableHttpStatusCodes = [408, 413, 429, 500, 502, 503, 504, 521, 522, 524];
```

Comment:
I find weird that we retry on server errors. Maybe "internal server" error, but "gateway" issue seems difficult to overcome.
Same for 413. If the payload is too huge, I am not sure it is a good idea to retry.

Reply:
i just have removed a lot of statusCodes, today's behavior is that they are ALL retried, so it's already a progress
but i agree with you tho, an ultimately target should to not even have this retry behavior on the network stack side imo